2.ML_model/2.DP_trained_model.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+jupyter nbconvert --to python DP_trained_model.ipynb
+python DP_trained_model.py
```

Comment:
On second thought, I think it is better to not specify any verb because the model is being trained, evaluated, and interpreted. I removed the verb in https://github.com/WayScience/phenotypic_profiling_model/pull/2/commits/5300c46a1686b52a32c77847c8cdf6aeaea814da. Let me know what you think.